Makefile

```diff
@@ -57,13 +57,13 @@ ftest: bin/pytest bin/elastic-ingest
 	connectors/tests/ftest.sh $(NAME) $(PERF8)
 
 run: install
-	bin/elastic-ingest --debug
+	bin/elastic-ingest
```

Comment:
do we want to put back `--debug`

Reply:
I'm not sure - I think it's okay to use the setting from configuration file now - if you want debug locally, you put debug locally, otherwise it's just INFO. 

No hard opinion on that though.